shared/src/vyxal/Interpreter.scala

```diff
@@ -354,4 +372,43 @@ object Interpreter:
     scribe.trace(s"Result of executing function: $res")
     res
   end executeFn
+
+  def createObject(con: VConstructor)(using ctx: Context): VObject =
+    val fields = ctx.globals.classes(con.name).fields
+    ctx.privatable += con.name
+    val assignedFields = mut.Map[String, (Visibility, VAny)]()
```

Comment:
I'd prefer a `Field` case class over `(Visibility, VAny)`, but I don't feel too strongly about it, so feel free to resolve this comment